Libraries/LibWeb/Painting/SVGSVGPaintable.cpp

```diff
@@ -101,7 +106,7 @@ void SVGSVGPaintable::paint_descendants(PaintContext& context, PaintableBox cons
 
     paintable.before_children_paint(context, PaintPhase::Foreground);
     paintable.for_each_child([&](auto& child) {
-        paint_svg_box(verify_cast<PaintableBox>(child));
+        paint_svg_paintable(child);
         return IterationDecision::Continue;
     });
```

Comment:
so, basically the problem is that we could encounter TextPaintable here and then verify cast will fail. then the fix could be much simpler here:
```diff
--- a/Libraries/LibWeb/Painting/SVGSVGPaintable.cpp
+++ b/Libraries/LibWeb/Painting/SVGSVGPaintable.cpp
@@ -100,8 +100,8 @@ void SVGSVGPaintable::paint_descendants(PaintContext& context, PaintableBox cons
     };

     paintable.before_children_paint(context, PaintPhase::Foreground);
-    paintable.for_each_child([&](auto& child) {
-        paint_svg_box(verify_cast<PaintableBox>(child));
+    paintable.for_each_child_of_type<PaintableBox>([&](PaintableBox& child) {
+        paint_svg_box(child);
         return IterationDecision::Continue;
     });
     paintable.after_children_paint(context, PaintPhase::Foreground);
```
FYI TextPaintable only serves for hit-testing purposes, so it's ok to simply skip it here.